gapic/src/main/com/google/gapid/server/GapisProcess.java

```diff
@@ -89,6 +89,11 @@ protected Exception prepare(ProcessBuilder pb) {
 
     String gapirFlags = "";
 
+    if (settings.reportCrashes) {
+      args.add("-crashreport");
+      args.add(settings.analyticsClientId);
```

Comment:
-crashreport is a boolean flag, is it not?

Reply:
Yes, good spot